odh-jupyter-trash-cleanup/odh_jupyter_trash_cleanup/trash.py

```
@@ -0,0 +1,57 @@
+    count = 0
+    if not p.exists():
+        return 0
+    # Refuse to traverse if the subdir itself is a symlink
+    if p.is_symlink():
+        self.log.warning("Refusing to traverse symlinked trash subdir: %s", p)
+        return 0
+    # If a file exists where a directory is expected, remove it
+    if p.is_file():
+        try:
+            p.unlink(missing_ok=True)  # type: ignore[arg-type]
+            return 1
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)